libs/shared/contentful/src/lib/queries/purchase-with-details-offering-content/util.ts

```diff
@@ -55,7 +57,7 @@ export class PurchaseWithDetailsOfferingContentUtil {
     return this.transformedPurchaseByPlanId[planId];
   }
 
-  get purchaseCollection() {
-    return this.rawResult.purchaseCollection;
+  get purchaseCollections() {
```

Comment:
Could this be changed back to purchaseCollection()? The logic to flatten the results into one purchaseCollection, instead of an array of purchaseCollections, should be handled by this function.